### PR TITLE
Remove all build folder when run build at root

### PR DESCRIPTION
**Pull request by dqtkien**
## Type

- [ ] Feature
- [ ] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [x] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "tsc --build tsconfig.json",
+    "build": "rm -rf packages/*/build && tsc --build tsconfig.json",
```
Comment (chiro-hiro) [inline package.json:8]: We need a script to clean all `node_modules` i experiences many problem with yarn symlink. And I prefer to have `clean build` script to clean `build`, `node_module` folders then install everything then build again.

Comment (dqtkien) [reply package.json]: I add `clean-build` script

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.json",
+    "clean-build": "find . -name 'node_modules' -type d -exec rm -rf {} + && find . -name 'build' -type d -exec rm -rf {} + && yarn && yarn build",
     "generate-docs": "spectaql spectaql-config.yml"
```
Comment (chiro-hiro) [inline package.json:11]:
```suggestion
    "clean": "find `pwd` -name 'node_modules' -type d -exec rm -rf {} + && find `pwd` -name 'build' -type d -exec rm -rf {} +" 
    "generate-docs": "spectaql spectaql-config.yml"
```

File: package.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
   ],
   "scripts": {
     "build": "tsc --build tsconfig.json",
+    "clean-build": "find . -name 'node_modules' -type d -exec rm -rf {} + && find . -name 'build' -type d -exec rm -rf {} + && yarn && yarn build",
```
Comment (chiro-hiro) [inline package.json]:
```suggestion
    "clean-build": "yarn clean && yarn && yarn build",
```

File: package.json [hunk 4]
```diff
@@ -5,7 +5,9 @@
     "packages/*"
   ],
   "scripts": {
+  	"clean": "find $(pwd) -name 'node_modules' -type d -exec rm -rf {} \\; || true && find $(pwd) -name 'build' -type d -exec rm -rf {} \\; || true",
```
Comment (chiro-hiro) [inline package.json:8]:
```suggestion
    "clean": "find $(pwd) -name 'node_modules' -type d -exec rm -rf {} \\; || true && find $(pwd) -name 'build' -type d -exec rm -rf {} \\; || true",
```

**Review by chiro-hiro — APPROVED** [review]

**Pull request merged**